docs/Helix.md

```diff
@@ -16,9 +16,24 @@ To run Helix tests for one particular test project:
 
 This will restore, and then publish all the test project including some bootstrapping scripts that will install the correct dotnet runtime/sdk before running the test assembly on the helix machine(s), and upload the job to helix.
 
+## Overview of the helix usage in our pipelines
+
+- Required queues: Windows10, OSX, Ubuntu1604 
+- Full queue matrix: Windows[7, 81, 10], Ubuntu[1604, 1804, 2004], Centos7, Debian[8,9], Redhat7, Fedora28, Arm64 (Win10, Debian9)
+
+aspnetcore-ci runs non quarantined tests against the required helix queues as a required PR check and all builds on all branches.
+
+aspnetcore-quarantined-tests runs only quarantined tests against the required queues only on master every 4 hours.
+
+aspnetcore-helix-matrix runs non quarantined tests against all queues twice a day only on master.
+
 ## How do I look at the results of a helix run on Azure Pipelines?
 
-There's a link embedded in the build.cmd log of the helix target on Azure Pipelines, near the bottom right that will look something like this:
+The easiest way to look a test failure is via the tests tab in azdo which now should show a summary of the errors and have attachements to the relevant console logs.
+
+You can also drill down into the helix web apis if you take the HelixJobId from the Debug tab of a failing test, and the HelixWorkItemName and go to: helix.dot.net/api/2019-06-17/jobs/<jobId>/workitems/<workitemname> which will show you more urls you can drill into for more info. 
```

Comment:
Suggest adding `https://` to the URI and guarding it with backticks to make sure it's not considered a real URI

Also likely worth mentioning:
- The https://helix.dot.net/ home page displays information about the available public queues (nothing about the related BYOC pools and queues or the internal Helix queues)
- Some superficial information about both BYOC and Helix agents is available at https://github.com/dotnet/arcade/blob/8ca46105193bd25c95af49bc6cd3604aaefec980/Documentation/AzureDevOps/AzureDevOpsOnboarding.md#agent-queues
- More detailed and always up-to-date information about all of the agents is available at https://helix.dot.net/api/2018-03-14/info/queues